pixl_dcmd/src/pixl_dcmd/_database.py

```diff
@@ -41,16 +41,15 @@ def get_uniq_pseudo_study_uid_and_update_db(
     project_slug: str, study_info: StudyInfo
 ) -> UID:
     """
-    Checks if record (slug, mrn, acc_num) exists in the database,
+    Checks if record (slug, study UID) exists in the database,
     gets the pseudo_study_uid if it is not None or records a new, unique one.
     Returns the pseudo_study_uid.
     """
     PixlSession = sessionmaker(engine)
     with PixlSession() as pixl_session, pixl_session.begin():
         existing_image = get_unexported_image(
             project_slug,
-            study_info.mrn,
-            study_info.accession_number,
+            study_info.study_uid,
```

Comment:
This method should check by study_uid, and if it can't find it, then search by mrn and accession number to match what we get from the VNA. At that point it throws an exception where it can't find based on either

Reply:
I started off doing that but then I figured that at this point all images would have a UID known to us, no? Because when we get a DICOM dataset from the VNA, even when queried using MRN + accession number, we can get the UID from the `StudyInstanceUID` field.
Or is it possible for a DICOM instance to lack the `StudyInstanceUID` field?

Reply:
Also not only that, but if its an emergency scan then the UID won't match whats in caboodle